lib/bamboo/test.ex

```diff
@@ -1,6 +1,8 @@
 defmodule Bamboo.Test do
   @timeout 100
```

Comment:
I think this should be configurable. Also should be renamed to `assert_timeout` since we have a separate timeout value for refute